composer.json

```diff
@@ -20,7 +20,7 @@
     },
     "require": {
         "php": "^7.4|^8.0",
-        "psr/log": "^1.1",
+        "psr/log": "^1.4|^2.0",
```

Comment:
Why 1.4 instead of 1.1?

Reply:
We cannot upgrade composer as `psr/log^ 2.0` requires `php ^ 8.0` :(